src/SFA.DAS.Commitments.Support.SubSite/Mappers/ApprenticeshipMapper.cs

```diff
@@ -0,0 +1,127 @@
+﻿using SFA.DAS.Commitments.Application.Queries.GetApprenticeshipsByUln;
+using SFA.DAS.Commitments.Domain.Entities;
+using SFA.DAS.Commitments.Support.SubSite.Extensions;
+using SFA.DAS.Commitments.Support.SubSite.Extentions;
+using SFA.DAS.Commitments.Support.SubSite.Models;
+using SFA.DAS.HashingService;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SFA.DAS.Commitments.Support.SubSite.Mappers
+{
+    public class ApprenticeshipMapper : IApprenticeshipMapper
+    {
+        private readonly IHashingService _hashingService;
+
+        public ApprenticeshipMapper(IHashingService hashingService)
+        {
+            _hashingService = hashingService;
+        }
+
+        public UlnSummaryViewModel MapToUlnResultView(GetApprenticeshipsByUlnResponse response)
+        {
+            return new UlnSummaryViewModel
+            {
+                Uln = response.Apprenticeships.First().ULN,
+                ApprenticeshipsCount = response.TotalCount,
+                SearchResults = response.Apprenticeships?.Select(o => MapToApprenticeshipSearchItemViewModel(o)).OrderBy(a => a.ApprenticeName).ToList()
```

Comment:
'Apprenticeships?' handles the case where Apprenticeships is null, but if Apprenticeships is null, 'response.Apprenticeships.First().ULN' would have already thrown.
We should either handle null Apprenticeships gracefully or not at all.